lib/hawkular/operations/operations_api.rb

```diff
@@ -222,18 +221,16 @@ def undeploy(hash, &callback)
     # Enable a WildFly deployment
     #
     # @param [Hash] hash Arguments for enable deployment
-    # @option hash [String]  :resource_path canonical path of the WildFly server from which to enable the deployment
+    # @option hash [String]  :resource_id ID of the WildFly server from which to enable the deployment
+    # @option hash [String]  :feed_id feed containing this resource
     # @option hash [String]  :deployment_name name of deployment to enable
     # @option hash [String]  :server_groups comma-separated list of server groups for the operation (default = ignored)
     #
     # @param callback [Block] callback that is run after the operation is done
     def enable_deployment(hash, &callback)
-      required = [:resource_path, :deployment_name]
+      required = [:resource_id, :feed_id, :deployment_name]
       check_pre_conditions hash, required, &callback
 
-      cp = ::Hawkular::Inventory::CanonicalPath.parse hash[:resource_path]
-      server_path = cp.up.to_s
-      hash[:resource_path] = server_path
```

Comment:
@jotak I suppose this is what you meant on the comments.

It confuses me that even the documentation stated that it required the Wildfly server (e.g. the documentation is the same now, or not?)

Reply:
I believed that it changed several times in the past and the doc wasn't updated... Not sure, I didn't check git history...

Reply:
Thanks, I'll check how is handled on MIQ

Reply:
I suppose we can receive the deployment_id and query the inventory for the server id?
That would ideally happen on hawkular-services if possible.
WDYT @jotak ?

Reply:
What do you have in memory on provider side, when this function is called?
If you have the whole Entity::Resource object that corresponds to the deployment, we can manage to add its parent_id directly in the model.
If you only have its ID, then yes we can add a REST endpoint to fetch parent.

Reply:
Done (both options: a rest endpoint + parent_id in model)